packages/arp/arp.3.1.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer: "Hannes Mehnert <hannes@mehnert.org>"
+authors: ["Hannes Mehnert <hannes@mehnert.org>"]
+homepage: "https://github.com/mirage/arp"
+doc: "https://mirage.github.io/arp/"
+dev-repo: "git+https://github.com/mirage/arp.git"
+bug-reports: "https://github.com/mirage/arp/issues"
+license: "ISC"
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {>= "2.7.0"}
+  "cstruct" {>= "6.0.0"}
+  "ipaddr" {>= "4.0.0"}
+  "macaddr" {>= "4.0.0"}
+  "logs"
+  "mirage-time" {>= "2.0.0"}
+  "lwt"
+  "duration"
+  "ethernet" {>= "3.0.0"}
+  "bisect_ppx" {dev & >= "2.5.0"}
+  "mirage-random" {with-test & >= "2.0.0"}
+  "mirage-random-test" {with-test & >= "0.1.0"}
+  "alcotest" {with-test}
+  "fmt" {with-test & >= "0.8.7"}
+  "mirage-vnetif" {with-test & >= "0.5.0"}
+  "mirage-clock-unix" {with-test & >= "3.0.0"}
+  "mirage-time-unix" {with-test & >= "2.0.0"}
+  "mirage-flow" {with-test & >= "2.0.1"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
Using local network
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test & os != "macos"}
```

Reply:
the tests do not use local network...

Reply:
The logs seem to say otherwise though
```
# tests.exe: [INFO] Connected Ethernet interface 02:50:00:00:00:01
# tests.exe: [INFO] Connected Ethernet interface 02:50:00:00:00:02
# tests.exe: [INFO] Sending gratuitous ARP for 192.168.3.10 (02:50:00:00:00:01)
# tests.exe: [DEBUG] starting arp listener for 02:50:00:00:00:01
# tests.exe: [INFO] Sending gratuitous ARP for 192.168.3.1 (02:50:00:00:00:02)
# tests.exe: [DEBUG] frame received for arpv4
# tests.exe: [DEBUG] Set IP for 02:50:00:00:00:02 to 192.168.3.1
# tests.exe: [DEBUG] Listener table contents after IP set on claimant: mac 02:50:00:00:00:01 ip 192.168.3.10 entries 2 timeout 800 retries 5
# 192.168.3.1 at 02:50:00:00:00:02 (timeout in 800)
# 192.168.3.10 at 02:50:00:00:00:01 (static advertising)
```

Reply:
These are virtual network interfaces (using mirage-vnetif).